int32 vid_idx
int32 img_idx
float32[] table_normal
float32[] table_midpoint
sensor_msgs/PointCloud2 cloud
int32[] pixel2voxel
sensor_msgs/Image img
sensor_msgs/Image mask